### PR TITLE
adds typings for the 'is-running' package.

### PR DESCRIPTION
**Pull request by danwbyrne**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/is-running/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Type definitions for is-running 2.1
+// Project: https://github.com/nisaacson/is-running
+// Definitions by: Daniel Byrne <https://github.com/danwbyrne>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function isRunning(pid: number): boolean;
```
Comment (armanio123) [inline types/is-running/index.d.ts]: It seems like the library not using `export default`, so this .d.ts file should not either.

Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html).

File: types/is-running/is-running-tests.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import isRunning from "is-running";
```
Comment (armanio123) [inline types/is-running/is-running-tests.ts]: You should use `import name = require('package');` instead.

If you still want to use the syntax `import name from 'package';` you need to use `esModuleInterop` on the compiler settings from your consuming code.

**Review by armanio123 — CHANGES_REQUESTED** [review]

File: types/is-running/index.d.ts (hunk 1 above)
Comment (danwbyrne) [reply types/is-running/index.d.ts]: Hi @armanio123, I finally had a chance to look at this more thoroughly this morning. I'm fairly new to making typings but I have a LOT more of them to submit so I want to make sure I'm getting this right from the beginning. 

In our library, we prefer to not use `require(...)` and from the changes I made to match the template like you suggest that seems to be the only way to import it. What makes setting it up as: 
```
export function isRunning(pid: number): boolean;
```
and then importing it like:
```
import { isRunning } from 'is-running';
```
not preferred here? Will we just have to modify the import on our end when we use it?

thanks in advance!

Comment (armanio123) [reply types/is-running/index.d.ts]: Not necessarily. You can still use the `import isRunning from 'is-running';` syntax, but you'll need to make sure that the compiler option `esModuleInterop` is enabled on your `tsconfig.json`. Details about the compiler options [here](https://www.typescriptlang.org/docs/handbook/compiler-options.html).

Also to make things right, the d.ts file should use `export =` syntax instead. Check this [documentation](https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#support-for-import-d-from-cjs-form-commonjs-modules-with---esmoduleinterop) for details.

This is how your d.ts file will look like:
```js
declare function isRunning(pid: number): boolean;
export = isRunning;
```

Comment (danwbyrne) [reply types/is-running/index.d.ts]: additionally should I make a similar adjustment to: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/toposort

it was already approved but I think it is in the same situation as this is.

Comment (armanio123) [reply types/is-running/index.d.ts]: Send me the PR. I'll be happy to check it out.

Comment (danwbyrne) [reply types/is-running/index.d.ts]: @armanio123 https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28399

**Review by armanio123 — APPROVED** [review]

**Pull request merged**